Sources/NostrSDK/Events/TextNoteEvent.swift

```diff
@@ -36,88 +36,26 @@ public final class TextNoteEvent: NostrEvent, CustomEmojiInterpreting {
         try super.init(kind: .textNote, content: content, tags: tags, createdAt: createdAt, signedBy: keypair)
     }
 
-    /// Pubkeys mentioned in the note content.
+    /// Pubkeys referenced in the note content.
+    @available(*, deprecated, message: "Deprecated in favor of referencedPubkeys. Mention is an overloaded term from NIP-10.")
```

Comment:
[NIP-01](https://github.com/nostr-protocol/nips/blob/master/01.md#tags) says
```
- The `p` tag, used to refer to another user: `["p", <32-bytes lowercase hex of a pubkey>, <recommended relay URL, optional>]`
```

Since this line uses the word `refer`, I've called the new property `referencedPubkeys`.